.travis.yml

```diff
@@ -31,4 +31,6 @@ install:
 
 script:
     - pip install virtualenv==1.10
+    - if [[ $TRAVIS_PYTHON_VERSION == 2* ]]; then pip install "mercurial>=3.3" ; fi
+    - if [[ $TRAVIS_PYTHON_VERSION == 2* ]]; then pip install python-hglib==1.5 ; fi
```

Comment:
I will add some notes -- `python-hglib` is being ported to Python 3, but isn't ready yet.  Mercurial is a longer term port; I wasn't sure how to indicate that Mercurial would need to come from system packages if it's a Python 3 test.